### PR TITLE
Add How to run a local dev node with Stylus support

### PR DESCRIPTION
**Pull request by TucksonDev**
This PR adds a page "How to run a local dev node with Stylus support" in the Stylus section. It also adds a notice in the regular local dev node how-to page to direct users toward the new page in case they are looking to have Stylus support.

Contents are very similar to "How to run a local dev node" page, with the assumption that Stylus-focused developers might not reach that page. For this reason, content has been added to a partial view, except for the initial 2 points. These points could also be in the partial view and pass the appropriate variable into it, although support is not very good for code blocks. [More information](https://docusaurus.io/docs/markdown-features/react#importing-markdown)

[Preview](https://nitro-docs-git-add-local-dev-node-stylus-i-8f42fd-offchain-labs.vercel.app/stylus/reference/local-stylus-dev-node)

### CODE REVIEW

File: arbitrum-docs/stylus/reference/local-stylus-dev-node.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+target_audience: 'Developers deploying smart contracts using Stylus'
+content-type: how-to
+sidebar_position: 4
+---
+
+import PublicPreviewBannerPartial from '../partials/_stylus-public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
+This how-to guides you through the process of setting up a local Arbitrum Nitro dev node, with Stylus support, to help you deploy and test smart contracts written in Solidity or any of the [Stylus supported languages](/stylus/stylus-gentle-introduction.md#in-a-nutshell) in a fully controlled environment. By following these steps, you'll deploy and run a full development environment on your local machine that includes a Nitro dev node (L2), a dev-mode geth parent chain (L1), and multiple instances with different roles.
```
Comment (amarrazza) [inline arbitrum-docs/stylus/reference/local-stylus-dev-node.mdx:16]:
```suggestion
This how-to guides you through the process of setting up a local Arbitrum Nitro dev node, with Stylus support, to help you deploy and test smart contracts written in Solidity or any [languages supported by Stylus](/stylus/stylus-gentle-introduction.md#in-a-nutshell) in a fully controlled environment. By following these steps, you'll deploy and run a full development environment on your local machine that includes a Nitro dev node (L2), a dev-mode geth parent chain (L1), and multiple instances with different roles.
```

Small suggestion to imply that even if there is no "official" support for a language via an SDK, users can still deploy programs in others

**Review by amarrazza — APPROVED** [review]
LGTM, depending on Mick's thoughts, might need a new folder in Stylus section for How-to's or can change some wording to fit this into a reference content type. I'm good with either or as is.

**Comment by TucksonDev** [comment]
I was hesitant to add a How-to section only for this page, but as you also mentioned it, I decided to add it. I can change it back if needed.

File: arbitrum-docs/node-running/how-tos/local-dev-node.mdx [hunk 2]
```diff
@@ -6,6 +6,12 @@ content-type: how-to
 
 A local Arbitrum Nitro dev node can help you deploy and test smart contracts in a fully controlled environment. This how-to walks you through the process of deploying and running a full development environment on your local machine that includes a Nitro dev node, a dev-mode geth L1, and multiple instances with different roles.
 
+:::info Stylus support
+
+This guide follows the process to setup a regular nitro local dev node without Stylus support. If you want to have Stylus support for your local development environment, [check out this guide](/stylus/how-tos/local-stylus-dev-node.mdx).
```
Comment (symbolpunk) [inline arbitrum-docs/node-running/how-tos/local-dev-node.mdx:11]:
```suggestion
If you want your local development environment to support Stylus, [check out this guide](/stylus/how-tos/local-stylus-dev-node.mdx).
```

**Review by symbolpunk — APPROVED** [review]

**Pull request merged**